doc/ansible/dev/advanced_options.md

```diff
@@ -50,3 +50,29 @@ You may want to manage what your operator watches and the owner references. This
 ```
 ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 ```
+
+## Max Workers
+
+Worker maximums can be set in two ways. Ansible-based operator **authors**
```

Comment:
WDYT about make it less verbose and remove the ` Ansible-based operator **authors**`? It is the Ansible guide/doc so all here will be about ansible. I think we could rewording to be more directly, I mean less verbose.